### DISCUSSION TITLE
全局热键中，“sendkeys”失效

### DISCUSSION THREAD

**Discussion by Einstein1103** [shuax/MouseInc.Settings#477] [Q&A]
![image](https://user-images.githubusercontent.com/62929210/209839265-4074354b-905c-43a1-a2d1-3b9a17abf158.png)
比如这个，按下alt+h后，只会执行后一步的动作，["SendKeys", "Ctrl+C"]不会被执行；
但是若将指令复制进鼠标手势中，就没有任何问题，很奇怪。

**Comment by hanson-reas** [comment]
太好了  我还以为只有我遇到这个问题了呢
